docs/services/jupyterlab.md

```diff
@@ -0,0 +1,104 @@
+[](){#ref-jlab}
+# JupyterLab
+
+## Access and setup
+
+The JupyterHub service enables the interactive execution of JupyterLab on [Daint][ref-cluster-daint] on a single compute node.
+
```

Comment:
Not a must, just thinking out loud: would it still make sense to add a `!!! note` saying something about jypyterhub on eiger?